component/elements.jsx

```diff
@@ -33,27 +48,24 @@ export const Color = Object.fromEntries(
 export const divider = <Color.Comment># {'··'.repeat(38)} #</Color.Comment>
 export const equal = <Color.Pink> = </Color.Pink>
 
-export const Main = ({ children }) => (
-  <main>
-    {toChildArray(children).map((child) => [
-      <div>{divider}</div>,
-      <section>{child}</section>,
-    ])}
-  </main>
-)
+export const Main = ({children}) => <div className="main-content">{children}</div>
```

Comment:
you use the `className` and `class` attributes in the same code. 
choose one convention name to name your class  attributes or props